study.md

```diff
@@ -54,7 +96,7 @@ already wrote and include the information about Dave's fish.
 Using the JSON your wrote above, can you write a method on the JSON?
 
 ```json
-<!-- your answer here -->
+json.parse("jason has")
```

Comment:
I was expecting a "no, you cant write a method in JSON" but I can see what you're going for here.  it would be JSON.parse(somejason) to turn it into a javascript object